src/packages/core/tests/server.test.ts

```diff
@@ -369,8 +425,7 @@ describe("server", () => {
         "-0o1",
         "-0b1",
         "0o",
-        "0b",
-        0
+        "0b"
```

Comment:
Yes, these are both an encoding of `0` in octal and binary, respectively, but we don't support that, and neither does node's `http.createServer.listen(<port>)`